packages/in_app_purchase/in_app_purchase_android/android/build.gradle

```diff
@@ -52,11 +52,11 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.annotation:annotation:1.0.0'
-    implementation 'com.android.billingclient:billing:3.0.2'
-    testImplementation 'junit:junit:4.12'
-    testImplementation 'org.json:json:20180813'
-    testImplementation 'org.mockito:mockito-core:3.6.0'
-    androidTestImplementation 'androidx.test:runner:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    implementation 'androidx.annotation:annotation:1.3.0'
+    implementation 'com.android.billingclient:billing:4.1.0'
```

Comment:
this will probably need more testing from our side, as well as exposing new APIs described in the changelog since 3.0.2

Reply:
in the version 3.0.7 continue this value 

implementation 'com.android.billingclient:billing:3.0.2'.